src/middleware/isnt-webhook.ts

```diff
@@ -4,14 +4,14 @@ import { IncomingMessage } from "http"
 
 export function isntWebhook(request: IncomingMessage, options: { path?: string }) {
   // GitHub sends all events as POST requests
-  if (request.method !== "POST") {
+  if (request.method !== "POST" || !request.url) {
```

Comment:
I don't think that the `!request.url` check belongs here, given the comment. We should only check for the method here. Check for the URL below